GraphBLAS/Source/Factories/GB_ops_template.h

```diff
@@ -858,7 +858,7 @@ inline void GB_FUNC (POW) (GB_Z_X_Y_ARGS)
     }
     inline void GB_FUNC (CMPLX) (GxB_FC32_t *z, const float *x, const float *y)
     {
-        #if defined ( __cplusplus ) || GB_COMPILER_MSC || defined (CMPLX)
+        #if defined ( __cplusplus ) || defined (GXB_HAVE_COMPLEX_MSVC) || defined (CMPLX)
```

Comment:
Maybe this would work instead, without the need for GXB_COMPILER_MSC or GXB_HAVE_COMPLEX_MSVC:

`#if defined ( __cplusplus ) || !defined (complex) || defined (CMPLX)`

since only the MSVC compiler has a complex.h without a `#define complex ...` macro.

Reply:
> since only the MSVC compiler has a complex.h without a #define complex ... macro.

That might be the case today. But afaict, there is no guarantee that this will also be the case for any future version of MSVC (or `clang-cl`).

Reply:
Checking for `!defined (complex)` would also not work in C++ code (where `complex` is not a preprocessor definition). So, there needs to be a different approach for `GraphBLAS.h` anyway.

Reply:
OK.  Line 890 needs to change too but I can take care of that after I merge it in.